src/components/inputs/select-input/select-input.js

```diff
@@ -70,14 +70,23 @@ export class SelectInput extends React.Component {
       option => (has(option, 'value') ? option : option.options)
     );
 
+    const selectedOptions = optionsWithoutGroups.filter(
+      option =>
+        props.isMulti
+          ? props.value.includes(option.value)
+          : has(option, 'value') && option.value === props.value
+    );
+
+    // We default to null, `type ValueType = OptionType | OptionsType | null | void`
+    /**
+     * we default to null
+     * > type ValueType = OptionType | OptionsType | null | void
+     * https://react-select.com/props#prop-types
+     */
```

Comment:
This comment looks a bit weird. Did you forget to delete the first part of it?

I would frame the comment more about the use-case we do this for (supporting changing a selected value back to nothing when `undefined` is passed / no related option exists).